modules/net-vpn-ha/variables.tf

```diff
@@ -107,6 +111,7 @@ variable "vpn_gateway_create" {
   description = "Create HA VPN Gateway. Set to null to avoid creation."
   type = object({
     description = optional(string, "Terraform managed external VPN gateway")
+    stack_type  = optional(string, "IPV4_ONLY") # IPV4_ONLY, IPV4_IPV6
```

Comment:
this should be an `ipv6 = optional(bool)` type attribute, it's an enum with just two possible values